docs/TutorVersionManager.rst

```diff
@@ -0,0 +1,124 @@
+#####################
+Tutor Version Manager
+#####################
+
+tvm is a version manager for tutor, this allows you to install and use different versions of tutor via the command line. 
+    
+
+Manage a tutor version
+-----------------------
+
+**Install**
+
+To install tutor you have to call ``tvm install v<TUTOR_VERSION>``
+
+.. code-block:: bash  
+    
+    tvm install v13.2.2
+    tvm install v12.2.0
+
+If you don't know which are the tutor versions, you can use ``tvm list``
+
+**Unisntall**
+
+To uninstall tutor you have to call ``tvm uninstall v<TUTOR_VERSION>``
+
+
+
+Check tutor versions
+--------------------
+
+If you want to list or check the installed versions use the command ``tvm list``, which shows you a complete list of all available tutor versions and highlights installed version(s) and the active one.
+
+.. image:: images/tvm_list.png
+    
+    
+**Flags**
+
+
+- Limit results to the latest versions ``--limit <number>`` or ``-l <number>``
+
+.. code-block:: bash
+    
+    tvm list -l 10
+    tvm list --limit 5
+
+
+Switching between tutor versions
+--------------------------------
+To switch between tutor versions you should call ``tvm use v<TUTOR_VERSION_INSTALLED>``
+
+.. code-block:: bash
+    
+    tvm use v12.2.0
+    tvm use v14.0.0
+ 
```

Comment:
I think that we need to specify that is necessary to install the versions before.